### PR TITLE
flask set up

### PR DESCRIPTION
**Pull request by garyfarb**
To start the flask application and the database, use the command 
`docker-compose -f docker-compose.yml up`

This will create the necessary containers and images. Once they have been created, log in to the database using the assigned credentials by visiting 
`http://localhost:8080/`

As a reminder, the credentials are:
UserName: dev
Password: 498capstone
Database: psychClinic

Once you have logged in, you can visit to see the Flask application in action
`http://localhost:5000/`
At the moment, the flask app just displays all the data in the database, this will have to get changed to fit our requirements later on so feel free to change that up.

### CODE REVIEW

File: app/app.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+from flask import Flask, render_template, jsonify
+import pymysql 
+
+app = Flask(__name__)
+
+# configuration for connection to the database
+db_config = {
```
Comment (cainsusk) [inline app/app.py:7]: we might want to put this in a "github secret" in the future

**Review by cainsusk — APPROVED** [review]
This looks great ! Thanks for setting up the python in docker !

**Comment by cainsusk** [comment]
also, maybe we should put the info for connecting to the backend in the README?

To update the readme, do `git commit --amend` and then `git push --force`

**Review by manda-02 — APPROVED** [review]
LGTM 🚀

**Comment by cainsusk** [comment]
> also, maybe we should put the info for connecting to the backend in the README?
> 
> To update the readme, do `git commit --amend` and then `git push --force`

You can also just make a new commit with the README changes.

It might also be good to change the `start_db` script to incorporate the python as well -- and maybe change the name to `start_app`

**Pull request merged**